posthog/hogql/database/database.py

```diff
@@ -92,12 +100,15 @@ def add_warehouse_tables(self, **field_definitions: Any):
             setattr(self, f_name, f_def)
 
 
+BUILT_IN_TABLE_NAMES = [attr for attr in dir(Database) if isinstance(getattr(Database, attr), Table)]
```

Comment:
This change to table names seems unrelated to the week start date and I don't really understand what it's for - maybe separate this out into a separate PR.

Reply:
Yes, definitely. It was also leftover from a previous HogQL-based approach